hw/top_earlgrey/data/ip/chip_spi_host_testplan.hjson

```diff
@@ -16,9 +16,81 @@
             - Run with min and max SPI clk frequencies and with single, dual and quad SPI modes.
 
             Verify all SPI host instances in the chip.
+
+            Notes for silicon targets:
+            - The testbench that mimics a physical SPI device should measure the clock to verify that the frequencies for each speed mode is correct.
+            This can be done by checking timing of the beginning and the end of the packet.
             '''
+      features: [
+        "SPIHOST.RATE.STANDARD",
+        "SPIHOST.RATE.DUAL",
+        "SPIHOST.RATE.QUAD",
+      ]
       stage: V2
+      si_stage: SV2
       tests: ["chip_sw_spi_host_tx_rx"]
     }
+    {
+      name: chip_sw_spi_host_pass_through
+      desc: '''Verify that the SPI host can be configured in pass through mode.
+
+            - Connect the SPI host to a serial NOR flash device.
+            - Configure an SPI device and that SPI host in pass through mode.
+            - Use the testbench's SPI host to write a word to the flash through the passthrough SPI device and host in OpenTitan.
+            - Read that word back.
+            - Check that the word read back is the same as the one written.
```

Comment:
Maybe it makes sense to be explicit about the fact that these two instructions are for the external SPI host, not something inside OT?